CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Is this necessary? IMO this is all common sense.

Reply:
Just removed it.